app/Community/Listeners/NotifyMessageThreadParticipants.php

```diff
@@ -141,6 +143,7 @@ private function forwardToDiscord(
                     //  - Original:  "Unwelcome Concept: Lots of Rings [12345] (Sonic the Hedgehog)"
                     //  - Formatted: "12345: Lots of Rings (Sonic the Hedgehog)"
                     if (preg_match('/^(Incorrect type:|Issue:|Unwelcome Concept:|Writing:)\s*(.*)\s*\[([0-9]+)\]\s*(\(.*\))$/', $messageThread->title, $titleMatches)) {
+                        $originalTitle = $messageThread->title;
```

Comment:
This doesn't look to be used.

Reply:
Looks like I missed that line. Removed in latest.